modules/model/shared/src/main/scala/engage/model/EngageCommand.scala

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package engage.model
+
+import cats.kernel.Eq
+
+sealed trait EngageCommand extends Product with Serializable
+
+object EngageCommand {
+
+  case object McsFollow   extends EngageCommand
+  case object ScsFollow   extends EngageCommand
+  case object CrcsFollow  extends EngageCommand
+  case object Pwfs1Follow extends EngageCommand
+  case object Pwfs2Follow extends EngageCommand
+  case object OiwfsFollow extends EngageCommand
+  case object AowfsFollow extends EngageCommand
+  case object Cwfs1Follow extends EngageCommand
+  case object Cwfs2Follow extends EngageCommand
+  case object Cwfs3Follow extends EngageCommand
+  case object Odgw1Follow extends EngageCommand
+  case object Odgw2Follow extends EngageCommand
+  case object Odgw3Follow extends EngageCommand
+  case object Odgw4Follow extends EngageCommand
+  case object McsPark     extends EngageCommand
+  case object ScsPark     extends EngageCommand
+  case object CrcsPark    extends EngageCommand
+  case object Pwfs1Park   extends EngageCommand
+  case object Pwfs2Park   extends EngageCommand
+  case object OiwfsPark   extends EngageCommand
+  case object AowfsPark   extends EngageCommand
+  case object Cwfs1Park   extends EngageCommand
+  case object Cwfs2Park   extends EngageCommand
+  case object Cwfs3Park   extends EngageCommand
+  case object Odgw1Park   extends EngageCommand
+  case object Odgw2Park   extends EngageCommand
+  case object Odgw3Park   extends EngageCommand
+  case object Odgw4Park   extends EngageCommand
+
+  implicit val engageCommandEq: Eq[EngageCommand] = Eq.fromUniversalEquals
+
+  implicit class EngageCommandOps(self: EngageCommand) {
```

Comment:
Maybe the `name` can be included in the `EngageCommand` trait rather than as an extra operation?